src/web-socket-handler.ts

```diff
@@ -137,11 +137,12 @@ export class WebSocketHandler implements WebSocketInterface {
         stdin: stream.Readable,
         streamNum: number = 0,
         retryCount: number = 3,
+        // kind of hacky, but otherwise we can't wait for the writes to flush before testing.
+        addFlushForTesting: boolean = false,
```

Comment:
Is this used anywhere? I don't see it anywhere in the new tests.

Reply:
It's used to grab the callback here:

https://github.com/kubernetes-client/javascript/pull/2259/files#diff-f289babd329db39c105b7cb4df60f7f510f4e8eb43d35e8907aea07061afc01fR489

Reply:
I parsed the code wrong when reading this. I was looking for a named parameter 🤦